web/pandas/community/blog/pandas-3.0-release-candidate.md

```diff
@@ -0,0 +1,125 @@
+  copies
+- **New `pd.col` syntax**: Initial support for `pd.col()` as a simplified syntax
+  for creating callables in `DataFrame.assign`
+
+You can find the complete list of changes in our
+[release notes](https://pandas.pydata.org/docs/dev/whatsnew/v3.0.0.html).
+
+## Important changes requiring code updates
+
+As a major release, pandas 3.0 includes some breaking changes that may require
+updates to your code. The two most significant changes are:
+
+### 1. Dedicated string data type by default
+
+Starting with pandas 3.0, string columns are automatically inferred as `str`
+dtype instead of the numpy `object` (which can store any Python object).
+
+**Example:**
+```python
+# Old behavior (pandas < 3.0)
+>>> ser = pd.Series(["a", "b"])
+>>> ser
+0    a
+1    b
+dtype: object  # <-- numpy object dtype
+
+# New behavior (pandas 3.0)
+>>> ser = pd.Series(["a", "b"])
+>>> ser.dtype
+>>> ser
+0    a
+1    b
+dtype: str  # <-- new string dtype
+```
+
+This change improves performance and type safety, but may require code updates,
+especially for library code that currently looks for "object" dtype when
+expecting string data.
+
+For more details, see the
+[migration guide for the new string data type](https://pandas.pydata.org/docs/dev/user_guide/migration-3-strings.html).
+
+### 2. Consistent copy/view behaviour with Copy-on-Write (CoW)
+
+Copy-on-Write is now the default and only mode in pandas 3.0. This makes
+behavior more consistent and predictable, but requires updates to certain coding
+patterns.
+
+The most impactfull change is that **chained assignment will no longer work**.
+As a result, the `SettingWithCopyWarning` is also removed (since there is no
+longer ambiguity whether it would work or not).
+
+**Example:**
+```python
+# Old behavior (pandas < 3.0) - chained assignment
+df["foo"][df["bar"] > 5] =   # This might modify df (unpredictable)
+
+# New behavior (pandas 3.0) - must do the modification in one step (e.g. with .loc)
+df.loc[df["bar"] > 5, "foo"] = 100
+```
+
+In general, any result of an indexing operation or method now always behaves as
+if it were a copy, so modifications of the result won't affect the original
+DataFrame.
+
+For more details, see the
+[Copy-on-Write migration guide](https://pandas.pydata.org/docs/dev/user_guide/copy_on_write.html#migrating-to-copy-on-write).
+
+
+## Call to Action: test the Release Candidate
+
+We need your help to ensure a smooth pandas 3.0 release!
+
+Especially if you have pandas code in production or maintain a library with
+pandas as a dependency, it is strongly recommended to run your test suites with
+the release candidate, and report any issue to our issue tracker before the
+official 3.0.0 release.
+
+How can you best test the release candidate?
+
+1. **First update to the latest released pandas 2.3** (if you are not already
+   running that version) and test it with your codebase. It is recommended to
+   resolve any deprecation warning before upgrading to pandas 3.0.
```

Comment:
Could also have setting `infer_string` and `copy_on_write` to True as steps 2 or 2/3, but don't feel strongly.